README.md

```diff
@@ -152,6 +153,15 @@ Ignore incoming errors and resolve like no errors occurred
 
 Return both the errors and data, only works with `rawRequest`.
 
+### IgnoreOperationName
+
+By default the GraphQLClient tries to extract the operationName from the document.
+You can define `ignoreOperationName` in the constructor of GraphQLClient to avoid the extraction process if it is not needed.
+
+```ts
+const client = new GraphQLClient(endpoint, { ignoreOperationName: true })
+```
+
```

Comment:
Could you explain the following:

- Example of document sent, ignored vs not ignored
- Why this matters

Links to other information sources are fine.

Reply:
I have made a new commit which I hope answers your questions. Feel free to ask me anything, I will be happy to answer.